README
This is the README file for the Paravel Theme package.

Paravel Theme is a simple KDE theme for Paravel Systems.  It includes
wallpaper and a custom KSplash theme.

(C) Copyright 2006,2008 Fred Gleason <fredg@paravelsystems.com>



ChangeLog
2008-08-23 Fred Gleason <fredg@paravelsystems.com>
	* Initial package created from 'srl_theme'.
2008-08-23 Fred Gleason <fredg@paravelsystems.com>
	* Updated graphics for Paravel Systems.
2008-08-23 Fred Gleason <fredg@paravelsystems.com>
	* Added 'make_slack.in', 'slack-desc.in' and 'slack_doinst.sh'.
2008-08-23 Fred Gleason <fredg@paravelsystems.com>
	* Fixed a path in 'paravel_theme.spec.in' that broke the splash
	screen under OpenSuSE.
2008-08-26 Fred Gleason <fredg@paravelsystems.com>
	* Updated the graphics in 'ksplash/Preview.png' and
	'ksplash/splash_top.png'.
	* Incremented the package version to 1.1.
2010-12-31 Fred Gleason <fredg@paravelsystems.com>
	* Updated installation paths for KSplash components for
	RedHat/CentOS 5.5.
2010-12-31 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 1.2.
2011-03-24 Fred Gleason <fredg@paravelsystems.com>
	* Added 'wallpapers/Paravel_Blue.png' and
	'wallpapers/Paravel_Gray1024.png'.
2011-03-24 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 1.3.
2011-03-25 Fred Gleason <fredg@paravelsystems.com>
	* Added 'wallpapers/rdairplay_skin.png' and
	'wallpapers/rdpanel_skin.png'.
2011-03-25 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 1.4.
2011-04-01 Fred Gleason <fredg@paravelsystems.com>
	* Added 'paravel_support.odt'.
2011-04-01 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 1.5.
2011-04-05 Fred Gleason <fredg@paravelsystems.com>
	* Updated 'paravel_support.odt'.
2011-04-05 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 1.6.
2011-04-05 Fred Gleason <fredg@paravelsystems.com>
	* Added 'broadcast_appliance_readme.odt'.
2011-04-06 Fred Gleason <fredg@paravelsystems.com>
	* Changed the fonts in 'paravel_support.odt' to Arial.
2011-04-05 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 1.7.
2013-03-27 Fred Gleason <fredg@paravelsystems.com>
	* Modified 'paravel_theme.spec.in' for CentOS 6.
	* Moved the wallpaper images from '/usr/share/wallpapers/' to
	'/usr/share/backgrounds/'.
	* Incremented the package version to 2.0.
2013-03-27 Fred Gleason <fredg@paravelsystems.com>
	* Added trigger logic to set default wallpaper in
	'paravel_theme.spec.in'.
2013-03-27 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 2.1.
2013-03-27 Fred Gleason <fredg@paravelsystems.com>
	* Updated 'broadcast_appliance_readme.odt'.
2013-04-01 Fred Gleason <fredg@paravelsystems.com>
	* Updated 'paravel_support.odt'.
2013-04-01 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 2.2.
2013-04-01 Fred Gleason <fredg@paravelsystems.com>
	* Added GConf defaults.
2013-04-01 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 2.3.
2013-04-01 Fred Gleason <fredg@paravelsystems.com>
	* Updated 'broadcast_appliance_readme.odt'.
2014-01-21 Fred Gleason <fredg@paravelsystems.com>
	* Added 'qt/qtrc'.
2014-01-21 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 2.4.
2014-05-15 Fred Gleason <fredg@paravelsystems.com>
	* Updated qt/qtrc to specify the polymer theme.
	* Incremented the package version to 2.5.
2014-08-16 Fred Gleason <fredg@paravelsystems.com>
	* Renamed 'configure.in' to 'configure.ac'.
	* Modified 'configure.ac' and 'Makefile.am' to use in-home
	RPM build tree.
2014-08-16 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 3.0.0.
2014-09-16 Fred Gleason <fredg@paravelsystems.com>
	* Added 'rddesktop_init.sh'.
	* Added 'rddesktop_init.desktop'.
2014-09-16 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 3.1.0.
2014-09-16 Fred Gleason <fredg@paravelsystems.com>
	* Added 'dconf-user'.
2014-09-16 Fred Gleason <fredg@paravelsystems.com>
	* Incremented the package version to 3.1.1.


autogen.sh
#!/bin/sh
##
##    (C) Copyright 2002-2006,2008 Fred Gleason <fredg@paravelsystems.com>
##
##    Adapted from './autogen.sh' in the Jack Audio Connection Kit.
##    Copyright (C) 2001-2003 Paul Davis, et al.
##
##    This program is free software; you can redistribute it and/or modify
##    it under the terms of version 2 of the GNU General Public License as
##    published by the Free Software Foundation;
##
##    This program is distributed in the hope that it will be useful,
##    but WITHOUT ANY WARRANTY; without even the implied warranty of
##    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
##    GNU General Public License for more details.
##
##    You should have received a copy of the GNU General Public License
##    along with this program; if not, write to the Free Software
##    Foundation, Inc., 59 Temple Place, Suite 330, 
##    Boston, MA  02111-1307  USA
##

aclocal $ACLOCAL_FLAGS || {
    echo "aclocal \$ACLOCAL_FLAGS where \$ACLOCAL_FLAGS= failed, exiting..."
    exit 1
}

automake --add-missing --foreign || {
    echo "automake --add-missing --foreign failed, exiting..."
    exit 1
}

autoconf || {
    echo "autoconf failed, exiting..."
    exit 1
}


rddesktop_init.sh
#!/bin/sh

#   rddesktop_init.sh
#
#   Do initial desktop setup for a user account.
#
#   (C) Copyright 2014 Fred Gleason <fredg@paravelsystems.com>
#
#   This program is free software; you can redistribute it and/or modify
#   it under the terms of the GNU General Public License version 2 as
#   published by the Free Software Foundation.
#
#   This program is distributed in the hope that it will be useful,
#   but WITHOUT ANY WARRANTY; without even the implied warranty of
#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#   GNU General Public License for more details.
#
#   You should have received a copy of the GNU General Public
#   License along with this program; if not, write to the Free Software
#   Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
#

#
# Set Wallpaper
#
gsettings set org.gnome.desktop.background picture-uri "$HOME/Pictures/Paravel_Gray1024.png"

#
# Desktop Lockdowns
#
# gsettings set org.gnome.desktop.lockdown disable-application-handlers true
gsettings set org.gnome.desktop.lockdown disable-lock-screen true
gsettings set org.gnome.desktop.lockdown disable-user-switching true

#
# Screensaver
#
gsettings set org.gnome.desktop.screensaver idle-activation-enabled false
gsettings set org.gnome.desktop.screensaver user-switch-enabled false

#
# Session Management
#
gsettings set org.gnome.SessionManager auto-save-session false
gsettings set org.gnome.SessionManager auto-save-session-one-shot false

#
# Cleanups
#
gsettings set org.gnome.desktop.privacy remove-old-temp-files true
gsettings set org.gnome.desktop.privacy remove-old-trash-files true

#
# Desktop Sounds
#
gsettings set org.gnome.desktop.sound event-sounds false

#
# Desktop Device Icons
#
gsettings set org.gnome.nautilus.desktop home-icon-visible true
gsettings set org.gnome.nautilus.desktop network-icon-visible false
gsettings set org.gnome.nautilus.desktop trash-icon-visible false
gsettings set org.gnome.nautilus.desktop volumes-visible false

#
# Remove Autostart Entry (so we only run once)
#
rm -f $HOME/.config/autostart/rddesktop_init.desktop


# End of rddesktop_init.sh


slack_doinst.sh
#!/bin/bash
#
# slack_doinst.sh
#
# paravel_theme installation package script for Slackware
#
# Copyright 2002-2006,2008 Fred Gleason <fredg@paravelsystems.com>
#
#      $Id: slack_doinst.sh,v 1.1 2008/08/24 02:05:38 cvs Exp $
#
#   This program is free software; you can redistribute it and/or modify
#   it under the terms of the GNU General Public License version 2 as
#   published by the Free Software Foundation.
#
#   This program is distributed in the hope that it will be useful,
#   but WITHOUT ANY WARRANTY; without even the implied warranty of
#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#   GNU General Public License for more details.
#
#   You should have received a copy of the GNU General Public
#   License along with this program; if not, write to the Free Software
#   Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
#


# End of slack_doinst.sh

